src/DGtal/io/viewers/Viewer3DFactory.h

```diff
@@ -192,6 +193,42 @@ namespace DGtal
     // StandardDSS6Computer
 
 
+    // naive3DDSSComputer
```

Comment:
Same argument as for your edits in Viewer3DFactory. Things should be duplicated.I think you may have to edit Board3DFactory (and 3Dto2D) as well